src/layers/XR3DLayer/xr-layer-fragment.glsl

```diff
@@ -129,7 +129,7 @@ void main(void) {
 		// Check if this point is on the "positive" side or "negative" side of the plane - only show positive.
 		float canShow = 1.;
 		for (int i = 0; i < _NUM_PLANES; i += 1) {
-			canShow *= max(0., sign(dot(normals[i], p - (distances[i] * normals[i]))));
+			canShow *= max(0., sign(dot(normals[i], p) + distances[i]));
```

Comment:
These two are equivalent I believe but probably cleaner to use the form described in the math.gl docs: https://math.gl/modules/culling/docs/api-reference/plane